tests/test_query.py

```diff
@@ -355,21 +314,12 @@ async def mock_get(
 @pytest.mark.parametrize(
     "invalid_image_modal", ["2nim:EEG", "apple", "some_thing:cool"]
 )
-def test_get_invalid_image_modal(test_app, invalid_image_modal, monkeypatch):
+def test_get_invalid_image_modal(
+    test_app, mock_invalid_get, invalid_image_modal, monkeypatch
```

Comment:
looks good, but we should try to have this kind of whitespace formatting be done as part of the linting so it'll be consistent. Could you start an issue @alyssadai to discuss what rules we want to have for the code formatting and how we enforce them?

Reply:
Aha, this formatting was actually applied by the linter! By the pre-commit hook for black specifically. At the moment, when a line of code surpasses a certain character count (I believe we set it to the PEP8 standard) it gets automatically reformatted like this prior to commit.